utils/geoip.go

```diff
@@ -31,31 +31,82 @@ var geoipFiles = map[string]string{
 	"GeoLite2-Country.mmdb": "http://geolite.maxmind.com/download/geoip/database/GeoLite2-Country.tar.gz",
 }
 
-// DownloadGeoIPDatabaseFiles into the target directory
-func DownloadGeoIPDatabaseFiles(dir string) error {
-	for filename, url := range geoipFiles {
+var legacyGeoipFiles = map[string]string{
+	"GeoIPASNum.dat": "http://download.maxmind.com/download/geoip/database/asnum/GeoIPASNum.dat.gz",
+	"GeoIP.dat":      "http://geolite.maxmind.com/download/geoip/database/GeoLiteCountry/GeoIP.dat.gz",
+}
+
+// GeoIPDir returns the geoip data dir for the given OONI Home
+func GeoIPDir(home string) string {
```

Comment:
Somethings weird about having these path things split up so that geoid is in util, the db folder is in `database/database.go`,
and the creation is in `/ooni.go`.

Consider a refactor to make this a bit easier to track down, potentially?

Reply:
Yeah maybe all the path stuff should go into a path package. It can't live in `ooni` or `database`, because of circular dependencies.